kit/service/transport.go

```diff
@@ -136,6 +136,10 @@ func (r *Router) MountRoute(route Route) *Router {
 		kithttp.ServerErrorEncoder(route.ErrorEncoder),
 		kithttp.ServerAfter(TraceIDSetter))
 
+	if route.CORSEnabled() {
```

Comment:
For some reason If I put `corsMiddleware` in `MountRoute` then it stops working. I tried on real service. It was in `Run` because it didn't work here.